Resources/public/js/views/fields/ez-country-view.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-country-view', function (Y) {
+    "use strict";
+    /**
+     * Provides the Country field view
+     *
+     * @module ez-country-view
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The country field view
+     *
+     * @namespace eZ
+     * @class CountryView
+     * @constructor
+     * @extends eZ.FieldView
+     */
+    Y.eZ.CountryView = Y.Base.create('countryView', Y.eZ.FieldView, [], {
+
+        /**
+         * Returns the name of the countries that will be used in the template.
+         * If a country is missing, it returns a warning
+         *
+         * @method _getFieldValue
+         * @protected
+         * @return {Array|String}
+         */
+        _getFieldValue: function () {
+            var that = this,
+                field = this.get('field'),
+                fieldValue = field.fieldValue,
+                isMultiple = this.get('fieldDefinition').fieldSettings.isMultiple,
+                res = [];
+
+            if (field && fieldValue) {
+                if (!isMultiple ) {
+                    return this._getCountry(fieldValue[0]);
+                }
+                Y.Array.each(fieldValue, function (country) {
+                    res.push(that._getCountry(country));
+                });
+            }
+            return res;
+        },
+
+        /**
+         * Returns the current country name.
+         * If the country is missing, it will return a warning and the country code.
```

Comment:
it does not *return* a warning but this is an implementation detail, you don't need to write that in the doc of the method.